examples/04_glm_first_level/plot_localizer_surface_analysis.py

```diff
@@ -64,51 +63,46 @@
 # We could use an individual :term:`mesh`,
 # but we first resort to a standard :term:`mesh`,
 # the so-called fsaverage5 template from the FreeSurfer software.
-import nilearn
 
-fsaverage = nilearn.datasets.fetch_surf_fsaverage()
 
 # %%
 # The projection function simply takes the :term:`fMRI` data and the mesh.
 # Note that those correspond spatially, are they are both in :term:`MNI` space.
 from nilearn import surface
-
-texture = surface.vol_to_surf(fmri_img, fsaverage.pial_right)
+from nilearn.experimental.surface import SurfaceImage, load_fsaverage
```

Comment:
just to make sure I understand:
- if we decide that the surface experimental is "ready" for the next release, we take it out of the `nilearn.experimental` package and adapt the examples and code accordingly
- otherwise we move this example to `examples/08_experimental` and keep along side the old version of this example?

Reply:
the ideal would be to move everything out of experimental now, but I'm afraid we're too short for the upcoming release ?